### ISSUE TITLE
GET /session/:sessionId/log/types throws exception

### ISSUE THREAD

**Issue by csnover** [selendroid/selendroid#365]
selendroid 0.9.0 on Android 19 (4.4)

Attempting to retrieve log types throws an exception.

```
INFO: return value from selendroid android server: 
org.json.JSONException: A JSONArray text must start with '[' at character 1
	at org.json.JSONTokener.syntaxError(JSONTokener.java:410)
	at org.json.JSONArray.<init>(JSONArray.java:113)
	at org.json.JSONArray.<init>(JSONArray.java:157)
	at io.selendroid.server.handler.GetLogTypes.handle(GetLogTypes.java:31)
	at io.selendroid.server.SelendroidServlet.handleRequest(SelendroidServlet.java:140)
	at io.selendroid.server.BaseServlet.handleHttpRequest(BaseServlet.java:70)
	at org.webbitserver.netty.NettyHttpControl.nextHandler(NettyHttpControl.java:78)
	at org.webbitserver.netty.NettyHttpControl.nextHandler(NettyHttpControl.java:62)
	at org.webbitserver.handler.PathMatchHandler.handleHttpRequest(PathMatchHandler.java:33)
	at org.webbitserver.netty.NettyHttpControl.nextHandler(NettyHttpControl.java:78)
	at org.webbitserver.netty.NettyHttpControl.nextHandler(NettyHttpControl.java:62)
	at org.webbitserver.handler.DateHeaderHandler.handleHttpRequest(DateHeaderHandler.java:21)
	at org.webbitserver.netty.NettyHttpControl.nextHandler(NettyHttpControl.java:78)
	at org.webbitserver.netty.NettyHttpControl.nextHandler(NettyHttpControl.java:62)
	at org.webbitserver.handler.ServerHeaderHandler.handleHttpRequest(ServerHeaderHandler.java:25)
	at org.webbitserver.netty.NettyHttpControl.nextHandler(NettyHttpControl.java:78)
	at org.webbitserver.netty.NettyHttpControl.nextHandler(NettyHttpControl.java:67)
	at org.webbitserver.netty.NettyHttpChannelHandler$2.run(NettyHttpChannelHandler.java:72)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:744)
```

**Comment by asm89** [comment]
I'm closing this issue due to inactivity.

If is a question rather than an issue, please send questions to the [selendroid user group](https://groups.google.com/forum/#!forum/selendroid).

 If you think it's an actual problem with selendroid and still experience the issue described on the latest release of selendroid feel free to file a new issue. If you do please provide a [concise reproducible test case](http://sscce.org/) and describe what results you are seeing and what results you expect.

**Comment by csnover** [comment]
Reproduction:

1. Start a Selendroid session
2. Call GET /session/:sessionId/log/types

Expected: Returns a list of log types
Actual: Error

Selendroid 0.15

**Comment by lukeis** [comment]
fixed by 639d8df4da3209b202d98ee81d92af56b8767606
